lib/emoji.js

```diff
@@ -854,7 +854,6 @@ var EMOJI_MAP = jEmoji.EMOJI_MAP = {
   "🕥": ["U+1F565", "clock face ten-thirty", "1f565", ["-", "-"], ["-", "-"], ["-", "-"], ["-", "-"]],
   "🕦": ["U+1F566", "clock face eleven-thirty", "1f566", ["-", "-"], ["-", "-"], ["-", "-"], ["-", "-"]],
   "🕧": ["U+1F567", "clock face twelve-thirty", "1f567", ["-", "-"], ["-", "-"], ["-", "-"], ["-", "-"]],
-  "🕧": ["U+1F567", "grinning face", "1f600", ["-", "-"], ["-", "-"], ["-", "-"], ["-", "-"]],
```

Comment:
https://github.com/neilk/emoji/blob/e9c6564ea79193f1ae960993d8f1b6798006b474/lib/emoji.js#L5

> This file auto create by `bin/create_emoji_js.py`.

you should change tpl file.